server/controllers/BookController.js

```diff
@@ -75,6 +77,42 @@ class BookController {
       return responseMessage(response, 500, { message: error.message });
     }
   }
+
+  /**
+ * Add a new book
+ * @name addBook
+ * @param {Object} request
+ * @param {Object} response
+ * @returns {JSON} object
+ */
+  static async getBorrowingStats(request, response) {
+    try {
+      const user = JSON.parse(JSON.stringify(request.userData));
+      const { id } = user;
+      let filter = { userId: id };
+      const { roleName } = user.UserRoles[0].Role;
+      if (roleName === 'admin' || roleName === 'superadmin') filter = {};
+      const { limit, page } = request.query;
+      const { count } = await LendingHistory.findAndCountAll({ where: filter });
+      const totalPages = Math.ceil(count / (limit || 10));
+      if ((page || 1) > totalPages) return responseMessage(response, 400, { status: 'failure', message: 'no book(s) found on this page' });
+      const offset = (limit || 10) * ((page || 1) - 1);
```

Comment:
the page is a string once is coming from request.query, u didn't convert page to be a number, 
like(+limit || 10) * ((+page || 1) - 1)
your message is not okay ///message: 'no book(s) found on this page' ///
you can use this book\'s instead of this book(s)

Reply:
> the page is a string once is coming from request.query, u didn't convert page to be a number,
> like(+limit || 10) * ((+page || 1) - 1)
> your message is not okay ///message: 'no book(s) found on this page' ///
> you can use this book's instead of this book(s)

It works that way without explicitly converting to integer. Javascript implicitly converts both of then and compare with gives exactly the same result as if you were using integers. For the message is just a way of saying 'no book' or 'no books' found on this page.